src/components/table/draggable-table-cell/draggable-table-cell.scss

```diff
@@ -0,0 +1,16 @@
+.draggable-table-cell {
+  width: 21px;
```

Comment:
The icon sizes to change, plus, that's too small for a touch screen, there should be some padding I think?

Maybe UX should look at this?